barretenberg/cpp/src/barretenberg/ultra_honk/ultra_honk.test.cpp

```diff
@@ -16,61 +16,67 @@
 
 using namespace bb;
 
-using ProverInstance = ProverInstance_<UltraFlavor>;
-using VerificationKey = UltraFlavor::VerificationKey;
-
-std::vector<uint32_t> add_variables(auto& circuit_builder, std::vector<bb::fr> variables)
-{
-    std::vector<uint32_t> res;
-    for (size_t i = 0; i < variables.size(); i++) {
-        res.emplace_back(circuit_builder.add_variable(variables[i]));
+// TODO(https://github.com/AztecProtocol/barretenberg/issues/1075): Make the tests run on UltraKeccakFlavor as well so
+// we have a means of checking issues without having to run the Solidity verifier contract.
+template <typename Flavor> class UltraHonkTests : public ::testing::Test {
```

Comment:
The diffs only concern turning this test suite to TYPED_TEST to run on both UltraFlavor and UltraKeccak